arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_2.2-das-configuration-options-storage.mdx

```diff
@@ -0,0 +1,10 @@
+A DAS can be configured to use one or more of four storage backends:
+
+- [AWS S3](https://aws.amazon.com/s3/) bucket
+- Files on local disk
+- [Badger](https://dgraph.io/docs/badger/) database on local disk
+- [IPFS](https://ipfs.tech/)
+
+If more than one option is selected, store requests must succeed to all of them for it to be considered successful, while retrieve requests only require one of them to succeed.
```

Comment:
These files were temporarily kept in case I needed to rescue them again (they were used to hold the common content between both DAS guides). I've now deleted them since they are not used anymore.
Thanks for the flag!